tests/show-config-fixtures/empty.txt

```diff
@@ -51,7 +51,7 @@ sources = []
 source = []
 
 [copy]
-targets = []
+target = []
```

Comment:
I think that should be still plural, as we have an array of values here. Although the array itself might make it clear, I think it's clearer to actually name it `targets` as well.

Reply:
Having singular or plural is always a tough decission... I agree that it can be targets here.
But `target = "target_profile"` also works (due to `serde_as::OneOrMany`). And the CLI also has this as option `--target` (description: `Target repository (can be specified multiple times)`).

So, the question is what is more strange:
- `[copy] targets = "my_target"`
- `[copy] target = ["my_target"`, "my_target2"]
- `rustic copy --targets=my_target --targets=my_target2`

Moreover I'd like to be consistent with other options where we basically get arrays but serde/clap also allows to handle single results.

Currently, we use singular in most places, IIRC.

Reply:
is changed to `targets` and `--target` in clap.